plugins/maciVoting/contexts/CoordinatorContext.tsx

```diff
@@ -44,8 +45,7 @@ async function makeCoordinatorServicePostRequest<T>(url: string, body: string):
 }
 
 export const CoordinatorProvider = ({ children }: { children: ReactNode }) => {
-  const [finalizeStatus, setFinalizeStatus] = useState<FinalizeStatus>("notStarted");
-
+  const { checkMergeStatus, checkIsTallied } = useMaci();
```

Comment:
I moved the `checkMergeStatus` and `checkIsTallied` functions to the Maci context (they are not calls to the coordinator but to the MACI contracts). I made myself sure to change it in every use of these functions and it is working correctly